site/app/templates/course/EditCourseMaterialsForm.twig

```diff
@@ -3,6 +3,17 @@
 {% block title %}Edit Course Materials{% endblock %}
 {% block body %}
     <div id="material-edit-form" class="flex-col flex-col-space" data-directory = "Check">
+    
+        <label id="file_path-label">
```

Comment:
New CSS Ids should use dashes instead of underscores, see 
https://submitty.org/developer/software_and_system_design/coding_style_guide/css

But if this touching older JS / HTML and it would be too hard to change disregard